content/statsServiceMap.md

```diff
@@ -7,7 +7,6 @@ title: Using the dataRetrieval Stats Service
 categories: Data Science
 tags: 
   - R
-  - dataRetrieval
```

Comment:
Yeah forgot about adding that back.  An automatic or user-defined 'image' header field would be nice too.